### PR TITLE
Bump to v4.3.0

### PR DESCRIPTION
**Pull request by mheon**
Release notes and backports for v4.3.0 release, plus the actual release bump.

```release-note
NONE
```

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat and anyone else who might want to double-check the release notes... PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,112 @@
 # Release Notes
 
+## 4.3.0
+### Features
+- A new command, `podman generate spec`, has been added, which creates a JSON struct based on a given container that can be used with the Podman REST API to create containers.
+- A new command, `podman update`, has been added,which makes changes to the resource limits of existing containers. Please note that these changes do not persist if the container is restarted ([#15067](https://github.com/containers/podman/issues/15067)).
+- A new command, `podman kube down`, has been added, which removes pods and containers created by the given Kubernetes YAML (functionality is identical to `podman kube play --down`, but it now has its own command).
+- The `podman kube play` command now supports Kubernetes secrets using Podman's secrets backend.
+- Systemd-managed pods created by the `podman kube play` command now integrates with sd-notify, using the `io.containers.sdnotify` annotation (or `io.containers.sdnotify/$name` for specific containers).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:9]: **integrate**, not integrates

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,112 @@
...
+- The `podman secret ls` command now accepts the `--quiet`/`-q` option.
+- The `podman secret inspect` command now accepts a new option, `--pretty`, to print output in human-readable format.
+- The `podman stats` command now accepts the `--no-trunc` option.
+- The `podman save` command now accepts the `--signature-policy` option ([#15869](https://github.com/containers/podman/issues/15869)).
+- The `podman pod inspect` command now allows multiple arguments to be passed. If so, it will return a JSON array of the inspected pods ([#15674](https://github.com/containers/podman/issues/15674)).
+- A series of new hidden commands have been added under `podman context` as aliases to existing `podman system connection` commands, to improve Docker compatibility.
+- The remote Podman client now supports proxying signals for attach sessions when the `--sig-proxy` option is set ([#14707](https://github.com/containers/podman/issues/14707)).
+
+### Changes
+- Duplicate volume mounts are now allowed, so long as source, destination, and options all match ([#4217](https://github.com/containers/podman/issues/4217)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:43]: How about `are now allowed **on the command line**`? The wording as it is suggests that the mounts themselves are duplicated.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,112 @@
...
+- The `podman generate kube` and `podman play kube` commands have been renamed to `podman kube generate` and `podman kube play` to group Kubernetes-related commands. Aliases have been added to ensure the old command names still function.
+- A number of Podman commands (`podman init`, `podman container checkpoint`, `podman container restore`, `podman container cleanup`) now print the user-inputted name of the container, instead of its full ID, on success.
+- When an unsupported option (e.g. resource limit) is specified for a rootless container on a cgroups v1 system, a warning message is now printed that the limit will not be honored.
+- The installer for the Windows Podman client has been improved.
+- The `--cpu-rt-period` and `--cpu-rt-runtime` options to `podman run` and `podman create` now print a warning and are ignored on cgroups v2 systems (cgroups v2 having dropped support for these controllers) ([#15666](https://github.com/containers/podman/issues/15666)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:48]: Could this be consolidated with the entry on line 46 (two lines above) about unsupported resource limits? (Maybe not, maybe the `rootless` is an important difference)

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,112 @@
...
+- Privileged containers running systemd will no longer mount `/dev/tty*` into the container ([#15878](https://github.com/containers/podman/issues/15878)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:49]: Ugh. This is actually a bug in the PR title itself, it irked me but I didn't fix it. Shame on me.

This should actually be `/dev/ttyN`, or `/dev/ttyX`, or "tty devices other than `/dev/tty`". The use of `*`, in a context where one will read it as a glob, suggests that `/dev/tty` itself will not be mounted.

**Comment by rhatdan** [comment]
LGTM

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,112 @@
...
+- Events for containers that are part of a pod now include the ID of the pod in the event.
+- SSH functionality for `podman machine` commands has seen a thorough rework, addressing many issues about authentication.
+- The `--network` option to `podman kube play` now allows passing `host` to set the pod to use host networking, even if the YAML does not request this.
+- The `podman inspect` command on containers now includes the digest of the image used to create the container.
+- Pods created by `podman play kube` are now, by default, placed into a network named `podman-kube`. If the `podman-kube` network does not exist, it will be created. This ensures pods can connect to each other by their names, as the network has DNS enabled.
+
+### Bugfixes
+- Fixed a bug where the `podman network prune` and `podman container prune` commands did not properly support the `--filer label!=` option ([#14182](https://github.com/containers/podman/issues/14182)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:57]: typo, **filter** not filer

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,5 +1,112 @@
...
+- Fixed a bug where the `podman kube generate` command added an unnecessary `Secret: null` line to generated YAML ([#15156](https://github.com/containers/podman/issues/15156)).
+- Fixed a bug where the `podman kube generate` command did not set `enableServiceLinks` and `automountServiceAccountToken` to false in generated YAML ([#15478](https://github.com/containers/podman/issues/15478) and [#15243](https://github.com/containers/podman/issues/15243)).
+- Fixed a bug where the `podman kube play` command did not properly handle CPU limits ([#15726](https://github.com/containers/podman/issues/15726)).
+- Fixed a bug where the `podman kube play` command did not respect default values for liveness probes ([#15855](https://github.com/containers/podman/issues/15855)).
+- Fixed a bug where the `podman kube play` command did not bind ports if `hostPort` was not specified but `containerPort` was not ([#15942](https://github.com/containers/podman/issues/15942)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:62]: I think this should be "...`containerport` was" (i.e. remove the "not"). But please double-check.

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -1,5 +1,112 @@
...
+- Fixed a bug where the `--filter "{{.Config.Healthcheck}}"` option to `podman image inspect` did not print the image's configured healthcheck ([#14661](https://github.com/containers/podman/issues/14661)).
+- Fixed a bug where the `podman volume create -o timeout=` option could be specified even when no volume plugin was in use.
+- Fixed a bug where the `podman rmi` command did not emit `untag` events when removing tagged images ([#15485](https://github.com/containers/podman/issues/15485)).
+- Fixed a bug where API forwarding with `podman machine` VMs on windows could sometimes fail because the pipe was not created in time ([#14811](https://github.com/containers/podman/issues/14811)).
+- Fixed a bug where the `podman pod rm` command could error if removal of a container in the pod was interrupted by a reboot.
+- Fixed a bug where the `exited` and `exec died` events for containers did not include the container's labels ([#15617](https://github.com/containers/podman/issues/15617)).
+- Fixed a bug where running Systemd containers on a systemd not using Systemd as PID 1 could fail ([#15647](https://github.com/containers/podman/issues/15647)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:71]: ...on a **system** not using Systemd (not "on a systemd")

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -1,5 +1,112 @@
...
+- Fixed a bug where Podman did not pass all necessary environment variables (including `$PATH`) to Conmon when starting containers ([#15707](https://github.com/containers/podman/issues/15707)).
+- Fixed a bug where the `podman events` command could function improperly when no events were present ([#15688](https://github.com/containers/podman/issues/15688)).
+- Fixed a bug where the `--format` flag to various Podman commands did not properly handle template strings including a newline (`\n`) ([#13446](https://github.com/containers/podman/issues/13446)).
+- Fixed a bug where Systemd-managed pods would kill every container in a pod when a single container exited ([#14546](https://github.com/containers/podman/issues/14546)).
+- Fixed a bug where the `podman generate systemd` command would generate incorrect yAML for pods created without the `--name` option.
```
Comment (edsantiago) [inline RELEASE_NOTES.md:76]: **YAML** (capitalization glitch)

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -1,5 +1,112 @@
...
+- Fixed a bug where memory limits reported by `podman stats` could exceed the maximum memory available on the system ([#15765](https://github.com/containers/podman/issues/15765)).
+- Fixed a bug where the `podman container clone` command did not properly handle environment variables whose value contained an `=` character ([#15836](https://github.com/containers/podman/issues/15836)).
+- Fixed a bug where the remote Podman client would not print the container ID when running the `podman-remote run --attach stdin` command.
+- Fixed a bug where the `podman machine list --format json` command did not properly show machine starting status.
+- Fixed a bug where automatic updates would not error when attempting to update a container with a non-fully qualified image name ([#15879](https://github.com/containers/podman/issues/15879)).
+- Fixed a bug where the `podman pod logs --latest` command could panic ([#15556](https://github.com/containers/podman/issues/15556)).
+- Fixed a bug where Podman could leave lingering network namespace mounts on the system if cleaning up the network failed for an unrelated reason.
```
Comment (edsantiago) [inline RELEASE_NOTES.md:87]: Not sure what the "for an unrelated reason" adds here; Unrelated to what? I'd suggest nuking it.

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -1,5 +1,112 @@
...
+- Fixed a bug where specifying an unsupported URI scheme for `podman system service` to listen at would result in a panic.
+- Fixed a bug where the `podman kill` command would sometimes not transition containers to the exited state ([#16142](https://github.com/containers/podman/issues/16142)).
+
+### API
+- Fixed a bug where the Compat DF endpoint reported incorrect reference counts for volumes ([#15720](https://github.com/containers/podman/issues/15720)).
+- Fixed a bug where the Compat Inspect endpoint for Networks where an incorrect network option was displayed, causing issues with `docker-compose` ([#15580](https://github.com/containers/podman/issues/15580)).
```
Comment (edsantiago) [inline RELEASE_NOTES.md:93]: Sentence does not parse. Maybe "a bug **in** the Compat..."? Or maybe something else needed?

File: RELEASE_NOTES.md [hunk 11]
```diff
@@ -1,5 +1,112 @@
...
+- The Libpod Restore endpoint for Containers now features a new query parameter, `pod`, to set the pod that the container will be restored into ([#15018](https://github.com/containers/podman/issues/15018)).
+- Fixed a bug where the REST API could panic while retrieving images.
+- Fixed a bug where a cancelled connection to several endpoints could, potentially, induce a memory leak.
```
Comment (edsantiago) [inline RELEASE_NOTES.md:96]: ", potentially," does not add anything; nuke it

**Review by edsantiago — CHANGES_REQUESTED** [review]
Whew. No major issues, but enough to (IMO) merit a re-push. Nice work.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (mheon) [reply RELEASE_NOTES.md:48]: Rootless is an important difference, so they cannot

**Comment by mheon** [comment]
Re-pushed with fixes

**Comment by mheon** [comment]
Restarted one test. Otherwise should be ready. @Luap99 @baude PTAL.

**Comment by edsantiago** [comment]
I restarted five, earlier. All of them search-related. We're going to be seeing that one a lot, I suspect.

**Comment by mheon** [comment]
The optional release test is failing, but we should be good to merge without it.

**Comment by rhatdan** [comment]
Does podman info show -dev?

**Comment by Luap99** [comment]
> Does podman info show -dev?

Yes because the release PR always contains at least the `bump to vX.Y.Z-dev` commit after the real release/tagged commit. The optional release test make no sense unless it starts actually checking the `bump to vX.Y.Z` commit. And even then it might still go wrong when @mheon or anybody else who tags a release tag the wrong commit by accident!

**Pull request merged**